src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,58 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ARRAY_LENGTH = 10;
+    private int sizeInt = 0;
```

Comment:
We should not specify the data type in the name
```suggestion
    private int size = 0;
```